tests/tracking/test_rest_tracking.py

```diff
@@ -42,6 +43,8 @@
 # Root directory for all artifact stores created during this suite
 SUITE_ARTIFACT_ROOT_DIR = tempfile.mkdtemp(suffix="artifacts", dir=SUITE_ROOT_DIR)
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
```suggestion
_logger = logging.getLogger(__name__)
```

Can we rename this to `_logger`?

Reply:
Done!